ui/components/component-library/modal-content/modal-content.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import {
+  BackgroundColor,
+  BorderRadius,
+  BLOCK_SIZES,
+} from '../../../helpers/constants/design-system';
+
+import Box from '../../ui/box/box';
+
+import { ModalContentProps, ModalContentSize } from './modal-content.types';
+
+export const ModalContent = ({
+  className = '',
+  children,
+  size = ModalContentSize.Sm,
+  width,
+  modalContentRef, // Would have preferred to forwardRef but it's not trivial in TypeScript. Will update once we have an established pattern
```

Comment:
Getting this error and have read a bunch of articles but nothing that seems like a valid pattern yet

![Screenshot 2023-03-22 at 12 17 29 PM](https://user-images.githubusercontent.com/8112138/227013329-515a2962-ce03-4468-95c1-3510436da253.png)

https://fettblog.eu/typescript-react-generic-forward-refs/